qiskit_rigetti/_qcs_backend.py

```diff
@@ -28,17 +28,6 @@
 from ._qcs_job import RigettiQCSJob
 
 
-def _remove_barriers(circuit: QuantumCircuit) -> None:
```

Comment:
Barrier handling has been moved into the `RigettiQCSJob` class, as there's no straightforward way to replace the instruction with a pragma here